hello_helpers/src/hello_helpers/hello_misc.py

```diff
@@ -626,3 +627,35 @@ def preprocess_gripper_trajectory(trajectory):
     gripper_index = trajectory.joint_names.index(present_gripper_joints[0])
     trajectory.joint_names[gripper_index] = 'stretch_gripper'
     return trajectory
+
+
+class LoopTimer:
```

Comment:
This class feels like it's been added to our libraries half a dozen times

maybe hello_helpers should be its own pip install able package for things like this